### DISCUSSION TITLE
New package: pie charts for SILE

### DISCUSSION THREAD

**Discussion by Omikhleia** [sile-typesetter/sile#1989] [Show and tell]
New 3rd-party package: [piecharts.sile](https://github.com/Omikhleia/piecharts.sile)
Available on [Luarocks](https://luarocks.org/modules/Omikhleia/piecharts.sile)

Some people will remember that it was part of an experiment I initiated in [Summer 2022](https://github.com/Omikhleia/omikhleia-sile-packages/pull/25) with SILE 0.12.x, but never completed... But these re-vamped pie charts are much nicer!
It was very funny to code and hack -- I hope it will find some users.

**Comment by Omikhleia** [comment]
Of course, devoted communities go [much farther](https://github.com/johannes-wolf/cetz) in the same time scale, or less...

**Comment by alerque** [comment]
I've been meaning to port my personal finance reports ([#plaintextaccounting](https://plaintextaccounting.org) via `hledger` of course) from LaTeX to SILE for years. I don't have any pretty charts just tables right now, but this might be the motivation I need to switch it over!
